testdata/nml_with_invalid_ids.nml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's invalid about these IDs? Can you maybe add a comment?

Reply:
The ids are invalid because the same id appears in different trees what is forbidden by webknossos.